cinema/views.py

```diff
@@ -118,12 +150,18 @@
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.CreateModelMixin,
+    mixins.ListModelMixin,
+    GenericViewSet
```

Comment:
The `OrderViewSet` should use `ModelViewSet` instead of `GenericViewSet` to ensure it supports the required actions (list and create) as per the task description.